addons/web/static/src/views/list/list_controller.js

```diff
@@ -142,24 +142,27 @@ export class ListController extends Component {
         usePager(() => {
             const list = this.model.root;
             const { count, hasLimitedCount, isGrouped, limit, offset } = list;
-            return {
-                offset: offset,
-                limit: limit,
-                total: count,
-                onUpdate: async ({ offset, limit }, hasNavigated) => {
-                    if (this.model.root.editedRecord) {
-                        if (!(await this.model.root.editedRecord.save())) {
-                            return;
+            if (!this.model.useSampleModel) {
```

Comment:
To minimize the diff and the risk of conflicts in forwardports
```suggestion
            if (this.model.useSampleModel) {
                return;
            }
```